### DISCUSSION TITLE
Ошибка скачивания страницы - unexpected EOF

### DISCUSSION THREAD

**Discussion by ghost** [lirix360/ReadmangaGrabber#58] [General]
![image](https://user-images.githubusercontent.com/52920235/177177311-296b92d7-2d9b-48bb-b6df-be9b3e5d9c4b.png)

**Comment by lirix360** [comment]
> ![image](https://user-images.githubusercontent.com/52920235/177177311-296b92d7-2d9b-48bb-b6df-be9b3e5d9c4b.png)

Адрес манги? Том, глава?
